integration/entrypoint.sh

```diff
@@ -245,13 +264,19 @@ function pull_remove_multiple_images {
     nerdctl --snapshotter nydus image pull "${WORDPRESS_IMAGE}"
     nerdctl --snapshotter nydus image pull "${TOMCAT_IMAGE}"
 
-    nerdctl --snapshotter nydus image rm "${TOMCAT_IMAGE}"
-    nerdctl --snapshotter nydus image rm "${JAVA_IMAGE}"
-    nerdctl --snapshotter nydus image rm "${WORDPRESS_IMAGE}"
+    nerdctl --snapshotter nydus create --rm --net none "${TOMCAT_IMAGE}"
```

Comment:
Why need to do `image pull` -> `create` -> `image rm` -> `create` ?

Reply:
Because nydusd is not started right after `image pull`. Nydusd is started when preparing the writable active snapshot as the uppermost layer. So we must create a container to start nydusd. Then to test if snapshotter's nydusd daemons management works well

Reply:
Ok, maybe we can add some comments to describe these step by step.

Reply:
comments all added